config/vendor-api-v1.yml

```diff
@@ -601,10 +601,8 @@ components:
           type: string
           maxLength: 10240
           nullable: true
-          description: About this candidate’s English language qualifications, if
-            English is not their main language
-          example: I have a degree in English Language and Literature from the University
-            of Munich
+          description: Details of this candidate's English language qualification, if English is not their main language
+          example: 'Name: TOEFL, Grade: 20, Awarded: 1999'
```

Comment:
I was wondering if this constitutes as a breaking change, as in we were initially returning free text, now we are still returning a string but it is actually structured data in the text 🤔 ?

Reply:
I did look into what this would change in practice. It previously relied on the english_qualification_details free text, and fell back on this new structured text. That means the only changes will be:
- Applications with the old free text (and no EFL qualification added) will no longer return a value here. 
- Things with the old free text and the new will now start returning the new structured EFL qualification text.

I checked and there are only 5 applications with both the old and new fields set (the latter case above), and they are not HEI applications so won't have been ingested on the API ever before.

Now I think about it, maybe we should switch around the fallback rather than turning it off completely, so that things with the old text and no EFL qualification set will still return something. The reason I didn't do this before it that Tribal are interested in the EFL qualification in particular, but we should probably favour backwards compatibility.

What do you think?

Reply:
I've now gone with the slightly more backwards compatible change, so that we aren't left with a bunch of old applications returning `nil` for the field where they didn't before.

I now don't think this is a breaking change because we have just added specificity to a field. That is, where there used to be data that could be in any format, we now return something more predictable.